internal/store/elasticsearch/discovery_search_repository.go

```diff
@@ -113,6 +113,7 @@ func (repo *DiscoveryRepository) buildQuery(cfg asset.SearchConfig) (io.Reader,
 	}
 	query = repo.buildFilterMatchQueries(query, cfg.Queries)
 	query = repo.buildFunctionScoreQuery(query, cfg.RankBy, cfg.Text)
+	highLight := repo.buildHighLightQuery(cfg)
 
```

Comment:
Let's simplify the search request body construction:

```go
	body, err := elastic.NewSearchRequest().
		Query(query).
		Highlight(highLight).
		MinScore(defaultMinScore).
		Body()
	if err != nil {
		return nil, fmt.Errorf("build query: new search request: %w", err)
	}

	return strings.NewReader(body), nil
```

we can get rid of the `searchQuery` struct type.

Reply:
done